### PR TITLE
Added Footer CTA | Updated Button paddings

### PR DESCRIPTION
**Pull request by ShreyaThapa**
- Added Footer CTA component
- Updated Buttons to add paddings based on Button size

### CODE REVIEW

File: components/base/FooterCTA.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// Props
+type ButtonProps = {
+  text: string;
+};
+type Props = { topic: string; description: string; button?: ButtonProps };
+
+function FooterCTA({ topic, description, button }: Props) {
+  return (
+    <Container>
+      <div>
```
Comment (Finholt) [inline components/base/FooterCTA.tsx]: Could we style this div to add some padding-right to iPad and up breakpoints and some padding-bottom to mobile breakpoints? There are certain screen widths where the text pushes right up against the button

File: components/base/FooterCTA.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <H2 isDark>{topic}</H2>
```
Comment (Finholt) [inline components/base/FooterCTA.tsx:59]: Looks like the margins on this standard h2 are bumping the text off of the vertical center of the component and adding a lot of space between the header and sub-header compared to the designs

File: components/base/FooterCTA.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  background-color: ${PURPLE_DARK};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  @media (${maxWidth(BREAKPOINT_MOBILE)}) {
+    padding: 10px;
+    flex-direction: column;
+  }
+  /* Text sizes decrease in smaller screens */
+  > div > h2 {
```
Comment (Finholt) [inline components/base/FooterCTA.tsx]: This definitely wouldn't be something to address in this ticket, but I wonder if we could update the typography components to take in a `customStyles` prop or something 🤔

File: components/base/FooterCTA.tsx (hunk 1 above)
Comment (ShreyaThapa) [reply components/base/FooterCTA.tsx]: Ah, good catch! 
For this, I ended up setting a `padding-right: 20px` on all screens except mobile.

File: components/base/FooterCTA.tsx (hunk 2 above)
Comment (ShreyaThapa) [reply components/base/FooterCTA.tsx:59]: I ended up unsetting the default margins, and added a bottom margin to the elements. This allows there to be a margin between the bottom text and the button on mobile too.

File: components/base/FooterCTA.tsx (hunk 3 above)
Comment (ShreyaThapa) [reply components/base/FooterCTA.tsx]: Yeah, that sounds good! I believe we want to revisit Typography later, so will add this as an action item there as well! Thanks, Connor!

**Review by Finholt — APPROVED** [review]

**Pull request merged**